### PR TITLE
correcting gcp credentials request

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
/hold
seeing this error using the changes:
```
oc get credentialsrequests openshift-image-registry-gcs -n openshift-cloud-credential-operator -oyaml
apiVersion: cloudcredential.openshift.io/v1
kind: CredentialsRequest
metadata:
  creationTimestamp: "2019-07-25T21:20:16Z"
  finalizers:
  - cloudcredential.openshift.io/deprovision
  generation: 1
  labels:
    controller-tools.k8s.io: "1.0"
  name: openshift-image-registry-gcs
  namespace: openshift-cloud-credential-operator
  resourceVersion: "2582"
  selfLink: /apis/cloudcredential.openshift.io/v1/namespaces/openshift-cloud-credential-operator/credentialsrequests/openshift-image-registry-gcs
  uid: ff258b17-af21-11e9-a90b-42010a000005
spec:
  providerSpec:
    apiVersion: cloudcredential.openshift.io/v1
    kind: GCPProviderSpec
    predefinedRoles:
    - roles/storage.admin
    - roles/iam.serviceAccountUser
  secretRef:
    name: installer-cloud-credentials
    namespace: openshift-image-registry
status:
  conditions:
  - lastProbeTime: "2019-07-25T21:21:32Z"
    lastTransitionTime: "2019-07-25T21:21:32Z"
    message: 'failed to grant creds: not all required service APIs are enabled'
    reason: CredentialsProvisionFailure
    status: "True"
    type: CredentialsProvisionFailure
  lastSyncGeneration: 0
  provisioned: false
```

**Comment by abhinavdahiya** [comment]
More info on the blocker right now:

https://github.com/openshift/cloud-credential-operator/pull/93

File: manifests/01-registry-credentials-request-gcs.yaml [hunk 1]
```diff
@@ -11,4 +11,7 @@ spec:
     namespace: openshift-image-registry
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
-    kind: GCSProviderSpec
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/storage.admin
+    - roles/iam.serviceAccountUser
```
Comment (abhinavdahiya) [inline manifests/01-registry-credentials-request-gcs.yaml]: As https://github.com/openshift/cloud-credential-operator/pull/93 merged
@coreydaley can you add `skipServiceCheck: true` ...

Comment (coreydaley) [reply manifests/01-registry-credentials-request-gcs.yaml]: Added in current push, ptal

**Pull request merged**